src/api/v2/agent-templates/handlers/generations-get.ts

```
@@ -0,0 +1,166 @@
+// Handler
+// ---------------------------------------------------------------------------
+
+export async function generationsGetHandler(req: Request, res: Response) {
+  const { generationId } = req.params;
+
+  // 1. Validate wait_ms
+  const waitMs = parseWaitMs(req.query.wait_ms);
+  if (waitMs === -1) {
+    res
+      .status(400)
+      .json({ error: "wait_ms must be a non-negative integer" });
```

ESLint

[error] Replace `⏎······.status(400)⏎······` with `.status(400)`

(prettier/prettier)